components/sidebar/sidebar_category/sidebar_category.tsx

```diff
@@ -219,7 +219,7 @@ export default class SidebarCategory extends React.PureComponent<Props, State> {
             return null;
         }
 
-        if (category.type !== CategoryTypes.DIRECT_MESSAGES && category.type !== CategoryTypes.CUSTOM && !channels?.length) {
+        if (category.type !== CategoryTypes.DIRECT_MESSAGES && category.type !== CategoryTypes.CUSTOM && category.type !== CategoryTypes.CHANNELS && !channels?.length) {
```

Comment:
Perhaps we could just change this to `category.type === CategoryTypes.FAVORITES && !channels?.length for simplicity? Otherwise, looks good.

Reply:
oops yes!